src/components/ModelNumber/ModelNumberComponent.jsx

```diff
@@ -13,46 +14,56 @@ const placeMakesInSemanticUIOptions = assetMakesList =>
     value: option.id
   }));
 
-const ModelNumberComponent = props => (
-  <div>
-    <Form onSubmit={props.handleSubmit}>
-      <label htmlFor="model-number" className="label-style">
-        Model Number
-        <InputFluid
-          name="model-number"
-          onChange={props.onAddModelNumber}
-          placeHolder="Enter Model Number"
+const ModelNumberComponent = (props) => {
+  if (props.isLoading) {
+    return (
+      <div className="loader-container" >
+        <LoaderComponent />
+      </div>
+    );
+  }
+
+  return (
+    <div>
```

Comment:
Remove the outer div. `<Form>` encapsulates the whole markup so the `<div>` is redundant